SWLOR.Game.Server/Service/BaseService.cs

```diff
@@ -1313,7 +1313,7 @@ public static int CalculateMaxFuel(Guid pcBaseID)
                                       if (x == null) return 0;
                                       var baseStructure = DataService.BaseStructure.GetByID(x.BaseStructureID);
                                       return baseStructure.Storage + x.StructureBonus;
-                                  }) * 0.01f;
+                                  }) * 0.1f;
```

Comment:
This is a 10% increase now? What do max numbers look like?

Reply:
Honestly. Didn't even field test these. 10% may be too much, this is for stronidium. Number are just a shot in the dark.

Reply:
If there's not really a compelling reason for the change, I'd recommend leaving it at 1% per Stronidium Bonus.

Reply:
Or alternatively, keep it consistent with the other one at 3% per bonus.